### PR TITLE
[drogon, trantor] Bump drogon to 1.9.2 and trantor to 1.5.16

### PR DESCRIPTION
**Pull request by an-tao**

### CODE REVIEW

**Comment by LilyWangLL** [comment]
Thanks for your PR, please resolve the CI pipeline failures https://github.com/microsoft/vcpkg/pull/36249/checks?check_run_id=20641138506.

**Comment by an-tao** [comment]
> Thanks for your PR, please resolve the CI pipeline failures https://github.com/microsoft/vcpkg/pull/36249/checks?check_run_id=20641138506.

Hi, I have no idea about the error with the failed CI, where can I find the complete logs?

**Comment by LilyWangLL** [comment]
You can download the failure logs from here: https://dev.azure.com/vcpkg/public/_build/results?buildId=98670&view=artifacts&pathAsName=false&type=publishedArtifacts.

**Comment by LilyWangLL** [comment]
All features passed with following triplets:
```
x86-windows
x64-windows
x64-windows-static
```

File: ports/trantor/portfile.cmake [hunk 1]
```diff
@@ -2,10 +2,11 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO an-tao/trantor
     REF "v${VERSION}"
-    SHA512 f08bba448229d8121534cade55b74ac0d9cf5deb1c1116794c21fee37f8f8c3b3d5e6618d0806942a9bcf5eb30c87879ddaf1104417fb3fe14de0827670e03e8
+    SHA512 14380d3781dabaff22041c4b9abcbfc591640bbb30a08badc9724ccc6ba817ab895774e37f24311f83b1bd139ad50d232d9eab510a4ede20bf2dff89cd308728
     HEAD_REF master
     PATCHES
         000-fix-deps.patch
+        001-uwp.patch
```
[the remark this answers is not in the document]
Comment (an-tao) [reply ports/trantor/portfile.cmake:9]: Yes, but I want release it in next version.

**Review by LilyWangLL — APPROVED** [review]

**Review by LilyWangLL — APPROVED** [review]

**Pull request merged**